ebos/eclbasevanguard.hh

```diff
@@ -617,6 +617,31 @@ public:
     {
         return centroids_;
     }
+
+    /*!
+     * \brief Get the number of cells inth global leaf grid view.
+     * \warn This is a collective operation that needs to be called
+     * on all ranks.
+     */
+    std::size_t noGlobalCells() const
+    {
+        const auto& grid = asImp_().grid();
+        if (grid.comm().size())
```

Comment:
That looks unexpected.  Is it supposed to include `== 0`?  If not, what am I missing?